Packs/CortexXpanse/Playbooks/Xpanse_-_Alert_Handler.yml

```diff
@@ -241,20 +241,216 @@ tasks:
     quietmode: 0
     isoversize: false
     isautoswitchedtoquietmode: false
+  "7":
+    continueonerrortype: ""
+    form:
+      description: Add one or more tags to the assets(s) associated with the current Cortex XSOAR incident.
+      expired: false
+      questions:
+      - defaultrows: []
+        fieldassociated: ""
+        gridcolumns: []
+        id: "0"
+        label: ""
+        labelarg:
+          simple: What tag(s) do you want to add to asset(s)?
+        options: []
+        optionsarg: []
+        placeholder: ""
+        readonly: false
+        required: true
+        tooltip: Specify a comma-separated list of tags you want to add to the asset(s) associated to the current Cortex XSOAR incident.
+        type: shortText
+      sender: Your SOC team
+      title: Determine tags to add to asset(s)
+      totalanswers: 0
+    id: "7"
+    ignoreworker: false
+    isautoswitchedtoquietmode: false
+    isoversize: false
+    message:
+      bcc:
+      body:
+      cc:
+      format: ""
+      methods: []
+      subject:
+      timings:
+        completeafterreplies: 1
+        completeaftersla: false
+        completeafterv2: true
+        retriescount: 0
+        retriesinterval: 0
+      to:
+    nexttasks:
+      '#none#':
+      - "9"
+    note: false
+    quietmode: 0
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: ""
+      id: 54995d00-c6c0-4549-8ded-319d5bc01e0e
+      iscommand: false
+      name: Determine tags to add to asset(s)
+      type: collection
+      version: -1
+      description: Add one or more tags to the assets(s) associated with the current Cortex XSOAR incident.
+    taskid: 54995d00-c6c0-4549-8ded-319d5bc01e0e
+    timertriggers: []
+    type: collection
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 1020
+        }
+      }
+  "9":
+    continueonerrortype: ""
+    id: "9"
+    ignoreworker: false
+    isautoswitchedtoquietmode: false
+    isoversize: false
+    nexttasks:
+      '#none#':
+      - "2"
+    note: false
+    quietmode: 0
+    scriptarguments:
+      asm_id_list:
+        complex:
+          accessor: xpanseassetids
+          root: incident
+          transformers:
+          - args:
+              chars:
+                value:
+                  simple: '[]'
+            operator: StripChars
+          - args:
+              limit: {}
+              replaceWith: {}
+              toReplace:
+                value:
+                  simple: '"'
+            operator: replace
+      tags:
+        complex:
+          accessor: Answers
+          root: Determine tags to add to asset
```

Comment:
Re-check that section and saw that you've changed the question a bit.
Please update this as well and check on your tenant that it works as expected.
```suggestion
          root: Determine tags to add to asset(s)
```

Reply:
I tested this and don't see what I would need to change
![2024-07-02_10-15-12](https://github.com/demisto/content/assets/32227961/9dc03db5-a07e-4a14-b841-e4957c52d8ad)

Reply:
was tested without your proposed change and it works

Reply:
I uploaded the pack to my server before my suggestion and the web survey didn't change
Maybe there were other suggestions you've added here without updating your server?
<img width="785" alt="image" src="https://github.com/demisto/content/assets/88268646/756d7bbc-a218-4897-bf49-ae34ed691615">
<img width="870" alt="image" src="https://github.com/demisto/content/assets/88268646/97c6e8ac-eae8-46ba-b522-f37170d126e4">

Reply:
I am testing with XSOAR 8.6
I believe that where a data collection task outputs to context is based of the "Web Survey Title":
![2024-07-02_15-45-32](https://github.com/demisto/content/assets/32227961/2ee92a5d-4a7f-4aaf-aa24-be0eb9282bb6)
However, for some reason, in testing that part of context isn't populated
![2024-07-02_15-46-11](https://github.com/demisto/content/assets/32227961/946c07a2-f538-4b29-a302-7326e00a3838)
![2024-07-02_15-42-18](https://github.com/demisto/content/assets/32227961/01038b24-c942-49d4-b4df-dc58e075573d)
I can remove the "(s)" from the "Web Survey Title" if that makes more sense, but if I continue with your change, I don't think it will work.

Reply:
[removed (s) in survey title](https://github.com/demisto/content/pull/35111/commits/61226ae3497d3e6c1e3a8e4197029b555e808e51)